.github/workflows/test.yml

```diff
@@ -49,6 +49,10 @@ jobs:
             odoo-version: 17.0
           - python-version: 3.11
             odoo-version: 17.0
+        traefik_version:
```

Comment:
When updating an old template, which value will take by default?

Reply:
I have tried both cases, using an old template and creating a new one. In both cases there do not seem to be any big changes regarding the Trafic labels defined for the Odoo service in prod.yaml. In fact it doesn't let me choose the Traefik version

![imagen](https://github.com/Tecnativa/doodba-copier-template/assets/2092475/763e6f86-1497-482f-b42e-08e0aa378c40)

Reply:
Hi @pleirb , to update a template using the changes from this PR, use the command `copier update --vcs-ref fix-ci-error --trust`. Regarding what you mentioned, @pedrobaeza , by default in an old template Traefik 2 will be selected. In fact, as it has been done until now, there is no real difference between selecting Traefik 1 or Traefik 2. This is how it has worked so far.

Reply:
> Hola .pleirb , para actualizar una plantilla usando los cambios de esta PR, utilice el comando `copier update --vcs-ref fix-ci-error --trust`. Con respecto a lo que ha mencionado, se @pedrobaezaseleccionará por defecto en una plantilla antigua Traefik 2. De hecho, como se ha hecho hasta ahora, no hay diferencia real entre la selección de Traefik 1 o Traefik 2. Así ha funcionado hasta ahora.

I took the following steps to work with this PR

```
git clone https://github.com/Tecnativa/doodba-copier-template.git
cd doodba-copier-template
git fetch origin pull/476/head:pr-476
git checkout pr-476
cd ..
copier copy ./doodba-copier-template ./project-16-prod --trust
```
I answered all the questions as I normally do and then applied the command you suggested.
```
cd project-16-prod
copier update --vcs-ref fix-ci-error --trust
```
But I am getting the following error:
`Destination repository is dirty; cannot continue. Please commit or stash your local changes and retry.`
I have tried with a new project and with one already in use. In both cases I get this error. What will I be doing wrong?